packages/commonwealth/server/controllers/server_proposal_methods/compound/compoundVersion.ts

```diff
@@ -24,36 +30,46 @@ type ContractAndVersion = {
  */
 async function deriveCompoundGovContractAndVersion(
   compoundGovAddress: string,
-  provider: providers.Web3Provider
+  provider: providers.Web3Provider,
 ): Promise<ContractAndVersion> {
   try {
     const contract = GovernorAlpha__factory.connect(
       compoundGovAddress,
-      provider
+      provider,
     );
     await contract.guardian();
     return { version: GovVersion.Alpha, contract };
   } catch (e) {
     try {
       const contract = GovernorBravoDelegate__factory.connect(
         compoundGovAddress,
-        provider
+        provider,
       );
       await contract.proposalCount();
       await contract.initialProposalId();
       return { version: GovVersion.Bravo, contract };
     } catch (e1) {
       try {
-        const contract = GovernorCompatibilityBravo__factory.connect(
+        const contract = GovernorCountingSimple__factory.connect(
           compoundGovAddress,
-          provider
+          provider,
         );
         await contract.COUNTING_MODE();
-        return { version: GovVersion.OzBravo, contract };
+        await contract.proposalVotes(1);
```

Comment:
To confirm:
- CountingSimple has BOTH COUNTING_MODE and proposalVotes
- Bravo has ONLY COUNTING_MODE and NOT proposalVotes
- OzGov has neither
?

Reply:
Yup